index.js

```diff
@@ -141,21 +142,23 @@ var scssContextParser = (function () {
         context.value = match[4].trim();
         context.scope = match[5] || 'private';
       }
-      if (lineNumberFor !== undefined) {
-        context.line = {
-          start : lineNumberFor(startIndex) + 1,
-          end : lineNumberFor(endIndex) + 1
-        };
-      }
     } else {
-      var codeStart = ctxCode.indexOf('{');
-      if (codeStart > 0) {
+      startIndex = ctxCode.indexOf('{');
```

Comment:
Yeah, I guess if code is formatted like that, the starting line number will be off by one. I wasn't sure how much we cared about that level of precision for the line numbers; the biggest issue is to have some semi-reasonable values in place so that SassDoc doesn't crash :-)

If it's important to have the line numbers be precisely accurate, I will work on an update to the PR to achieve that. (I may need to handle trailing whitespace at the end of the code string better for that too.)